### PR TITLE
Added option to build against NVIDIA debug libraries

### PR DESCRIPTION
**Pull request by lilleyse**
This is a continuation of a really old PR https://github.com/CesiumGS/cesium-omniverse/pull/63. At the time we couldn't figure out how to use NVIDIA debug libraries and fell back to using the release versions.

While investigating a crash today I decided to revisit this in the hope of getting a more useful stack trace. It turns out we were actually very close, the main thing missing was needing to fetch the debug version of kit from packman.

So now if you want to build and run with debug libraries:

**Windows**

```cmake
# Build
cmake -B build -D CESIUM_OMNI_USE_NVIDIA_DEBUG_LIBRARIES=ON
cmake --build build --config Debug --target install --parallel 8

# Run
./extern/nvidia/_build/target-deps/kit-sdk-debug/kit.exe apps/cesium.omniverse.dev.kit
```

**Linux**

```cmake
# Build
cmake -B build -D CMAKE_BUILD_TYPE=Debug -D CESIUM_OMNI_USE_NVIDIA_DEBUG_LIBRARIES=ON
cmake --build build --target install --parallel 8

# Run
./extern/nvidia/_build/target-deps/kit-sdk-debug/kit apps/cesium.omniverse.dev.kit
```

Just be aware that it's a lot slower than using the normal release build of kit, which is why `CESIUM_OMNI_USE_NVIDIA_DEBUG_LIBRARIES` is OFF by default.

https://github.com/CesiumGS/cesium-omniverse/pull/372 and https://github.com/CesiumGS/cesium-omniverse/pull/373 were discovered while working on this. Those PRs should be merged first.

### CODE REVIEW

File: docs/developer-setup/README.md
Comment (lilleyse) [inline docs/developer-setup/README.md:35]: I removed these steps since they aren't actually required for building and running Cesium for Omniverse.

I think there should be a separate section that goes over how to load the extension in Code or USD Composer but it could be a separate PR.

File: docs/developer-setup/README.md
Comment (lilleyse) [inline docs/developer-setup/README.md:210]: 99% percent of the time we want to build **and** install, so I added `--target install` here.

File: docs/developer-setup/README.md
Comment (mattelser) [inline docs/developer-setup/README.md:54]: thanks for fixing this while you were in here!

File: docs/developer-setup/README.md
Comment (mattelser) [inline docs/developer-setup/README.md:373]: "install location" is somewhat overloaded, can this be more specific? like `/path/to/git/repo`?

File: docs/developer-setup/README.md
Comment (mattelser) [inline docs/developer-setup/README.md:381]: same as above

**Review by mattelser — COMMENTED** [review]
one small comment that shouldn't block approval, so pending successful test LGTM

File: docs/developer-setup/README.md
Comment (lilleyse) [reply docs/developer-setup/README.md:373]: `install` is the right term from the CMake / make / build system perspective. I don't think `/path/to/git/repo` is quite right because the installed files are usually placed somewhere else (e.g. how `genStubs.sh` is installing to `/temp`)

Comment (mattelser) [reply docs/developer-setup/README.md:373]: oh, I think I confused myself. You're totally right

**Comment by mattelser** [comment]
unfortunately crashed on linux. Here's the dump file:

[9e9738d9-515f-4260-6a81e0a9-02450489.dmp.zip](https://github.com/CesiumGS/cesium-omniverse/files/11900193/9e9738d9-515f-4260-6a81e0a9-02450489.dmp.zip)

**Comment by lilleyse** [comment]
It's crashing on linux for me as well.

Even if I run `./extern/nvidia/_build/target-deps/kit-sdk-debug/kit` by itself it crashes, so it's not related to our extension. It seems like the linux debug build is missing some cached shaders and isn't able to rebuild them.

I think this PR is ok to merge still since those problems are out of our control.

```
2023-06-29 16:37:40 [5,978ms] [Error] [gpu.foundation.plugin] Failed opening shader cache version file: /home/slilley/packman-repo/chk/kit-sdk/104.2+release.275.6d591d71.tc.linux-x86_64.debug/extscore/omni.rtx.shadercache/cache/shadercache/version
2023-06-29 16:37:40 [5,978ms] [Error] [gpu.foundation.plugin] Failed to process shader cache version file

```
```
2023-06-29 16:38:57 [83,841ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/database/DirtyPageCopy.compute.hlsl'.
2023-06-29 16:38:57 [83,841ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/database/DirtyElementsCopy.compute.hlsl'.
2023-06-29 16:38:57 [83,841ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/database/DirtyPageCopy.compute.hlsl
2023-06-29 16:38:57 [83,841ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/database/DirtyElementsCopy.compute.hlsl
2023-06-29 16:38:57 [83,841ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,841ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,894ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/scenedb/SparseMemCopy.compute.hlsl'.
2023-06-29 16:38:57 [83,894ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/scenedb/SparseMemCopy.compute.hlsl
2023-06-29 16:38:57 [83,894ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,894ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/utils/PrefixScanTexture.compute.hlsl'.
2023-06-29 16:38:57 [83,894ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/utils/PrefixScanTexture.compute.hlsl'.
2023-06-29 16:38:57 [83,894ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/utils/PrefixScanBuffer.compute.hlsl'.
2023-06-29 16:38:57 [83,894ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/utils/PrefixScanTexture.compute.hlsl
2023-06-29 16:38:57 [83,894ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/utils/PrefixScanTexture.compute.hlsl'.
2023-06-29 16:38:57 [83,894ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/utils/PrefixScanBuffer.compute.hlsl'.
2023-06-29 16:38:57 [83,894ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/utils/PrefixScanTexture.compute.hlsl
2023-06-29 16:38:57 [83,894ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,894ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/utils/PrefixScanBuffer.compute.hlsl
2023-06-29 16:38:57 [83,894ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/utils/PrefixScanTexture.compute.hlsl
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/utils/PrefixScanBuffer.compute.hlsl
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/lighting/EnvironmentPreprocessingLatLong.compute.hlsl'.
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/lighting/EnvironmentPreprocessingLatLong.compute.hlsl'.
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/lighting/EnvironmentPreprocessingLatLong.compute.hlsl'.
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/lighting/EnvironmentPreprocessingLatLong.compute.hlsl
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/lighting/EnvironmentPreprocessingLatLong.compute.hlsl'.
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/lighting/EnvironmentPreprocessingLatLong.compute.hlsl'.
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/lighting/EnvironmentPreprocessingLatLongConversion.compute.hlsl'.
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/lighting/EnvironmentPreprocessingLatLong.compute.hlsl
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/lighting/EnvironmentPreprocessingLatLongConversion.compute.hlsl'.
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/lighting/EnvironmentPreprocessingLatLong.compute.hlsl
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/lighting/EnvironmentPreprocessingLatLong.compute.hlsl
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/utils/PrefixScanBuffer.compute.hlsl'.
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/lighting/EnvironmentPreprocessingLatLongConversion.compute.hlsl
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/lighting/EnvironmentPreprocessingLatLong.compute.hlsl'.
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/lighting/EnvironmentPreprocessingLatLongConversion.compute.hlsl'.
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/lighting/EnvironmentPreprocessingLatLong.compute.hlsl
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/utils/PrefixScanBuffer.compute.hlsl
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/lighting/EnvironmentPreprocessingLatLong.compute.hlsl'.
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/lighting/EnvironmentPreprocessingLatLong.compute.hlsl
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/lighting/EnvironmentPreprocessingLatLongConversion.compute.hlsl
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/lighting/EnvironmentPreprocessingLatLongConversion.compute.hlsl'.
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,895ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/lighting/EnvironmentPreprocessingLatLongConversion.compute.hlsl
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/lighting/EnvironmentPreprocessingLatLong.compute.hlsl
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/filters/LowPass.cs.hlsl'.
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/lighting/EnvironmentPreprocessingLatLongConversion.compute.hlsl
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/filters/LowPass.cs.hlsl
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/lighting/EnvironmentPreprocessingApproxIBL.cs.hlsl'.
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/lighting/EnvironmentPreprocessingApproxIBL.cs.hlsl'.
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/lighting/EnvironmentPreprocessingApproxIBL.cs.hlsl'.
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/lighting/EnvironmentPreprocessingApproxIBL.cs.hlsl
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/lighting/EnvironmentPreprocessingApproxIBL.cs.hlsl
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/lighting/EnvironmentPreprocessingApproxIBL.cs.hlsl'.
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/lighting/EnvironmentPreprocessingApproxIBL.cs.hlsl
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/lighting/EnvironmentPreprocessingApproxIBL.cs.hlsl
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,896ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,897ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/scenedb/TranslateInstancePosition.compute.hlsl'.
2023-06-29 16:38:57 [83,897ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/scenedb/TranslateInstancePosition.compute.hlsl
2023-06-29 16:38:57 [83,897ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,897ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/scenedb/ValidateSbt.compute.hlsl'.
2023-06-29 16:38:57 [83,897ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/scenedb/ValidateSbt.compute.hlsl
2023-06-29 16:38:57 [83,897ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,915ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/scenedb/ProceduralAabb.comp.hlsl'.
2023-06-29 16:38:57 [83,915ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/scenedb/ProceduralAabb.comp.hlsl
2023-06-29 16:38:57 [83,916ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,916ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/scenedb/Skinning.comp.hlsl'.
2023-06-29 16:38:57 [83,916ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/scenedb/SkinNormal.comp.hlsl'.
2023-06-29 16:38:57 [83,916ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/scenedb/Skinning.comp.hlsl
2023-06-29 16:38:57 [83,916ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/scenedb/SkinNormal.comp.hlsl
2023-06-29 16:38:57 [83,916ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/scenedb/NormalGeneration.comp.hlsl'.
2023-06-29 16:38:57 [83,916ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,916ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,916ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/scenedb/NormalGeneration.comp.hlsl
2023-06-29 16:38:57 [83,916ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/scenedb/VertexPulling.comp.hlsl'.
2023-06-29 16:38:57 [83,916ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:38:57 [83,917ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/scenedb/VertexPulling.comp.hlsl
2023-06-29 16:38:57 [83,917ms] [Error] [gpu.foundation.plugin] PsoCompute::createPipeline and shader loading failed. RtxResult: 3
2023-06-29 16:39:00 [86,378ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/rasterizing/Point.pixel.hlsl'.
2023-06-29 16:39:00 [86,378ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/rasterizing/Line.vertex.hlsl'.
2023-06-29 16:39:00 [86,378ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/rasterizing/Point.geometry.hlsl'.
2023-06-29 16:39:00 [86,378ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/rasterizing/Line.geometry.hlsl'.
2023-06-29 16:39:00 [86,378ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/rasterizing/Line.geometry.hlsl'.
2023-06-29 16:39:00 [86,378ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/rasterizing/Line.pixel.hlsl'.
2023-06-29 16:39:00 [86,378ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/rasterizing/Point.geometry.hlsl'.
2023-06-29 16:39:00 [86,378ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/rasterizing/Line.vertex.hlsl'.
2023-06-29 16:39:00 [86,378ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/rasterizing/Point.geometry.hlsl
2023-06-29 16:39:00 [86,378ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/rasterizing/Line.geometry.hlsl'.
2023-06-29 16:39:00 [86,378ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/rasterizing/Line.geometry.hlsl
2023-06-29 16:39:00 [86,378ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/rasterizing/Line.geometry.hlsl
2023-06-29 16:39:00 [86,378ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/rasterizing/Point.geometry.hlsl
2023-06-29 16:39:00 [86,378ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/rasterizing/Point.pixel.hlsl
2023-06-29 16:39:00 [86,378ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/rasterizing/Line.pixel.hlsl
2023-06-29 16:39:00 [86,378ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/rasterizing/Line.vertex.hlsl
2023-06-29 16:39:00 [86,378ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/rasterizing/Line.geometry.hlsl
2023-06-29 16:39:00 [86,378ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/rasterizing/Line.vertex.hlsl
2023-06-29 16:39:00 [86,388ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/system/GenerateMipMap.comp.hlsl'.
2023-06-29 16:39:00 [86,388ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/system/GenerateMipMap.comp.hlsl
2023-06-29 16:39:00 [86,389ms] [Error] [gpu.foundation.plugin] Failed to async compile compute shader: rtx/system/GenerateMipMap.comp.hlsl
2023-06-29 16:39:00 [86,399ms] [Error] [gpu.foundation.plugin] Failed to compile compute shader: rtx/system/GenerateMipMap.comp.hlsl
2023-06-29 16:39:00 [86,400ms] [Error] [gpu.foundation.plugin] Failed to compile compute shader: rtx/system/GenerateMipMap.comp.hlsl
2023-06-29 16:39:00 [86,400ms] [Error] [gpu.foundation.plugin] Failed to compile compute shader: rtx/system/GenerateMipMap.comp.hlsl
2023-06-29 16:39:00 [86,401ms] [Error] [gpu.foundation.plugin] Failed to compile compute shader: rtx/system/GenerateMipMap.comp.hlsl
2023-06-29 16:39:00 [86,401ms] [Error] [gpu.foundation.plugin] Failed to compile compute shader: rtx/system/GenerateMipMap.comp.hlsl
2023-06-29 16:39:00 [86,402ms] [Error] [gpu.foundation.plugin] Failed to compile compute shader: rtx/system/GenerateMipMap.comp.hlsl
2023-06-29 16:39:00 [86,402ms] [Error] [gpu.foundation.plugin] Failed to compile compute shader: rtx/system/GenerateMipMap.comp.hlsl
2023-06-29 16:39:00 [86,414ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/raytracing/SimpleMiss.ms.hlsl'.
2023-06-29 16:39:00 [86,414ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/raytracing/SimpleMiss.ms.hlsl
2023-06-29 16:39:00 [86,416ms] [Error] [gpu.foundation.plugin] Cannot load shader file 'rtx/lightspeed/ReLAX/PackInputData.cs.hlsl'.
2023-06-29 16:39:00 [86,416ms] [Error] [gpu.foundation.plugin] buildShaderAsync() failed: rtx/lightspeed/ReLAX/PackInputData.cs.hlsl
2023-06-29 16:39:00 [86,416ms] [Error] [rtx.denoising.plugin] Failed to async compile compute shader: rtx/lightspeed/ReLAX/PackInputData.cs.hlsl
```

**Review by mattelser — APPROVED** [review]
agreed, issues outside our codebase shouldn't hold this up. approved

**Comment by lilleyse** [comment]
Also, it seems to be working in Kit 105

**Pull request merged**
